### DISCUSSION TITLE
can't examine database in sqlitebrowser or tableplus app

### DISCUSSION THREAD

**Discussion by ElegantSoft** [pocketbase/pocketbase#430] [Q&A] [answered]
I tried to open `pb_data` folder and open each file in tableplus or any database explorer and didn't show anything

**Comment by ganigeorgiev** [comment, marked as answer]
The PocketBase db file usually is located in `./pb_data/data.db` and there is no reason why you shouldn't be able to explore it with a 3rd party tool. You can even directly access it with the native sqlite3 cli: `sqlite3 pb_data/data.db`.

_I'll move the issue to Q&A discussion._
